Lab 1/Lab1.ipynb

```diff
@@ -2,11 +2,539 @@
+      "text/plain": [
+       "<Figure size 1080x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "fig, ax = plt.subplots(1, 1)\n",
+    "ax.hist(d,50, density=True)\n",
+    "plt.tick_params(labelsize = 24)\n",
+    "# plt.xlim([2.95,3.05])\n",
+    "x = np.linspace(3.2, 4, 1000)\n",
+    "ax.plot(x,stats.chi2.pdf(x, 55, loc = 3., scale = 0.01),linewidth = 8,alpha = 1)\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "628aa982-184b-4111-a4ad-7538bd66af7b",
+   "metadata": {},
+   "source": [
+    "#### 3A. \n",
+    "I choose a value of measurement of 3.72."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "7163c1b0-39e7-4c70-bf69-ed30c0ee917f",
+   "metadata": {},
+   "source": [
+    "#### 3B. \n",
+    "Let's propose a scientific question.\n",
+    "\n",
+    "__What is the probability that 3.72 is a measurement of the signal or of a greater signal?__"
```

Comment:
Explain what is meant by "signal" here. Why is it significant if the measurement is of "greater signal"?